src/libertem/analysis/gridmatching.py

```diff
@@ -1,20 +1,14 @@
 import numpy as np
 
-from libertem.utils import make_polar, make_cartesian
-
-
-def fastmatch(centers, refineds, peak_values, peak_elevations, zero, a, b, parameters):
-    corr = CorrelationResult(centers, refineds, peak_values, peak_elevations)
-    return Match.fastmatch(corr, zero, a, b, parameters)
-
-
-def affinematch(centers, refineds, peak_values, peak_elevations, indices):
-    corr = CorrelationResult(centers, refineds, peak_values, peak_elevations)
-    return Match.affinematch(corr, indices)
-
 
 class CorrelationResult:
```

Comment:
In practice, a CorrelationResult is used internally since users should use the functions of the Matcher classes, which accept ndarrays of the CorrelationResult properties and return Match objects. What does that imply, should I change something?